.azure-pipelines/generate-docs.yaml

```
@@ -20,7 +20,69 @@ resources:
       ref: refs/heads/master
 
 stages:
+  - stage: Check_Changes
+    jobs:
+    - job: docs_modified
```

Unify with other jobs in pipeline
```suggestion
    - job: detect_docs_change
      displayName: Check docs changes
```